framework/src/modules/bft/module.ts

```diff
@@ -49,6 +60,26 @@ export class BFTModule extends BaseModule {
 	// eslint-disable-next-line @typescript-eslint/no-empty-function
 	public async afterGenesisBlockExecute(_context: GenesisBlockExecuteContext): Promise<void> {}
 
-	// eslint-disable-next-line @typescript-eslint/no-empty-function
-	public async afterBlockExecute(_context: BlockAfterExecuteContext): Promise<void> {}
+	public async afterBlockExecute(context: BlockAfterExecuteContext): Promise<void> {
+		const votesStore = context.getStore(this.id, STORE_PREFIX_BFT_VOTES);
+		const paramsStore = context.getStore(this.id, STORE_PREFIX_BFT_PARAMETERS);
+		const paramsCache = new BFTParametersCache(paramsStore);
+		const bftVotes = await votesStore.getWithSchema<BFTVotes>(EMPTY_KEY, bftVotesSchema);
+
+		insertBlockBFTInfo(bftVotes, context.header, this._maxLengthBlockBFTInfos);
+		await paramsCache.cache(
+			bftVotes.blockBFTInfos[bftVotes.blockBFTInfos.length - 1].height,
+			bftVotes.blockBFTInfos[0].height,
+		);
+		await updatePrevotesPrecommits(bftVotes, paramsCache);
+		await updateMaxHeightPrevoted(bftVotes, paramsCache);
+		await updateMaxHeightPrecommitted(bftVotes, paramsCache);
+		updateMaxHeightCertified(bftVotes, context.header);
+		await votesStore.setWithSchema(EMPTY_KEY, bftVotes, bftVotesSchema);
+		const minRemovingHeight = Math.min(
+			bftVotes.blockBFTInfos[bftVotes.blockBFTInfos.length - 1].height,
+			bftVotes.maxHeightCertified,
+		);
+		await deleteBFTParameters(paramsStore, minRemovingHeight + 1);
```

Comment:
I also updated the BFT module LIP regarding this step, see https://github.com/LiskHQ/lips/pull/113/commits/27fb914c19327c0e5ed7bb9e8d32ba888beb26b6

Here it should be 
```
const minRemovingHeight = Math.min(
    bftVotes.blockBFTInfos[bftVotes.blockBFTInfos.length - 1].height,
    bftVotes.maxHeightCertified+1,
);
await deleteBFTParameters(paramsStore, minRemovingHeight);
```

I would also rename `minRemovingHeight` to `minHeightBFTParametersRequired` or something similar.